bugsnag-android-core/src/main/java/com/bugsnag/android/Error.java

```diff
@@ -98,4 +98,16 @@ static List<Error> createError(@NonNull Throwable exc,
                                    @NonNull Logger logger) {
         return ErrorInternal.Companion.createError(exc, projectPackages, logger);
     }
-}
+
+    @Nullable
+    public Stackframe addStackframe(@NonNull StackTraceElement element) {
+        return impl.addStackframe(element, logger);
+    }
+
+    @Nullable
+    public Stackframe addStackframe(@NonNull String method,
+                                    @NonNull String file,
+                                    @NonNull Number lineNumber) {
```

Comment:
These arguments should be `@Nullable` (as they are in the `Stackframe` constructor)